app/pages/nonsipp/sharesdisposal/TotalSharesNowHeldPage.scala

```diff
@@ -1,31 +0,0 @@
-/*
- * Copyright 2023 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package pages.nonsipp.sharesdisposal
-
-import config.Refined.{Max50, Max5000}
-import models.SchemeId.Srn
-import pages.QuestionPage
-import play.api.libs.json.JsPath
-import utils.RefinedUtils._
-
-case class TotalSharesNowHeldPage(srn: Srn, shareIndex: Max5000, disposalIndex: Max50) extends QuestionPage[Int] {
```

Comment:
@ccatosdevelopment this was unused, i.e never properly populated, wasn't it?

Reply:
Correct - it was created in advance to be used for the last page in the Shares Disposal journey, but when that ticket was picked up, another page was made and used instead of this one - so this page had no usages.